hapi-fees/src/main/java/com/hedera/services/usage/token/TokenUpdateUsage.java

```diff
@@ -142,7 +142,7 @@ public FeeData get() {
 		long txnBytes = newMutableRb + BASIC_ENTITY_ID_SIZE + noRbImpactBytes(op);
 		usageEstimator.addBpt(txnBytes);
 		if (op.hasTreasury()) {
-			addTokenTransfersRecordRb(1, 2);
+			addTokenTransfersRecordRb(1, 2, 1);
```

Comment:
We have a problem here. On TokenUpdate, when the treasury account is being changed, for fungibles we are transferring the balance to the new treasury. For nonfungibles, we do not have efficient way to do it.

For now we can do
```
addTokenTransfersRecordRb(1, 2, 0);
```
since we do not add any NFT transfer records during Token Update

Reply:
Corrected!